styles.css

```diff
@@ -683,7 +683,8 @@ main {
   .top-fram .bb {
     width: 19.2rem;
     height: 5.6rem;
-    padding: auto;
+
+    /* padding: auto; */
```

Comment:
- [ ] Kindly avoid leaving comments in your code. It makes your code unclean :+1:  You can check out this [thread](https://questions.microverse.org/t/is-writing-down-comments-on-the-code-a-good-practice/687) to read more about this :+1: